File: week_1_Bonrevoir_Objc/Bonrevoir/Bonrevoir/ViewController.m
//
//  ViewController.m
//  Bonrevoir
//
//  Created by Nicolas Salleron on 18/09/2017.
//  Copyright © 2017 Nicolas Salleron. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *labelText;


@end

@implementation ViewController

bool change = false;

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)labelChange:(id)sender {
   
    if(!change){
        [_labelText setText:@"Bonjour !!!"];
        [(UIButton*) sender setTitle:@"Dis au revoir !" forState:UIControlStateNormal];
        change = true;
    }else{
        [_labelText setText:@"Au revoir !!!"];
        [(UIButton*) sender setTitle:@"Dis bonjour !" forState:UIControlStateNormal];
        change = false;
    }
   
}


@end

File: week_1_Bonrevoir_Switf/Bonrevoir/Bonrevoir/ViewController.swift
//
//  ViewController.swift
//  Bonrevoir
//
//  Created by Nicolas Salleron on 18/09/2017.
//  Copyright © 2017 Nicolas Salleron. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBOutlet weak var labelDis: UILabel!
    @IBOutlet weak var buttonChange: UIButton!
    var change = false;
    @IBAction func changeLabel(_ sender: Any) {
        
        if(!change){
            labelDis.text = "Bonjour !!!"
            buttonChange.setTitle("Dis au revoir !", for: UIControlState.normal)
            change = true;
        }else{
            labelDis.text = "Au revoir !!!"
            buttonChange.setTitle("Dis bonjour !", for: UIControlState.normal)
            change = false;
        }
        
        
        
        
    }
}

